### PR TITLE
Update groups-filter-sort.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Updated filtering section to remove "Select" constraint.

### CODE REVIEW

File: source/boards/groups-filter-sort.rst [hunk 1]
```diff
@@ -34,11 +34,11 @@ To ungroup cards on table view, select the **Group by** option at the top of the
 Filters
 -------
 
-You can filter cards on your board if they utilize the **Select** property.
+You can filter cards on your board using all the text properties.
```
Comment (justinegeffen) [inline source/boards/groups-filter-sort.rst:37]: @wuwinson, is this accurate and should I also mention Filters now support all text properties! You’re no longer limited to only using filters with the Select property type. You can filter your board more granularly to see the cards most relevant to you. Boards now also supports filtering by the following property types:
- Text
- Email
- Phone
- URL

Comment (wuwinson) [inline source/boards/groups-filter-sort.rst:37]:
```suggestion
You can filter cards on your board if they utilize any of the following property types:
- Select
- Text
- Email
- Phone
- URL
```

File: source/boards/groups-filter-sort.rst [hunk 2]
```diff
@@ -34,11 +34,11 @@ To ungroup cards on table view, select the **Group by** option at the top of the
...
 
 .. note:: 
   
-   You must have at least one **Select** property on your board for filtering to work. Filtering by other property types such as **Person** or **Date** is not yet supported.
+   You must have at least one **Select** property on your board for filtering to work.
```
Comment (wuwinson) [inline source/boards/groups-filter-sort.rst:41]:
```suggestion
   You must have at least one of the property types listed above on your board for filtering to work. Filtering by other property types such as **Person** or **Date** is not yet supported.
```

File: source/boards/groups-filter-sort.rst (hunk 1 above)
Comment (wuwinson) [reply source/boards/groups-filter-sort.rst:37]: Hey Justine, users can still filter by the *Select* property as well, so many some tweaks here.

**Review by wuwinson — APPROVED** [review]

**Review by cwarnermm — APPROVED** [review]
Good updates! Really like @wuwinson's clarifications as well. Approving for expediency.

**Pull request merged**